### ISSUE TITLE
License for package metadata exposed through NuGet feeds

### ISSUE THREAD

**Issue by russcam** [NuGet/NuGetGallery#4495]
Is there any license for the package metadata provided by Nuget feeds? Specifically, the package _metadata_ returned from 

https://www.nuget.org/api/v2/Packages

I've looked everywhere on the NuGet site but no licensing is mentioned for this, only for the NuGet server source code (and for individual licenses applied to the _contents_ of packages).

[Is package metadata covered under the same Apache 2 license as the source code?](https://github.com/NuGet/NuGetGallery/blob/master/LICENSE.txt)

**Comment by skofman1** [comment]
Hi @russcam, you can get the license URL as provided by the package author from "LicenseUrl" property in OData.
For example: https://www.nuget.org/api/v2/Packages(id='newtonsoft.json',version='10.0.3')
LicenseUrl is <d:LicenseUrl>https://raw.github.com/JamesNK/Newtonsoft.Json/master/LICENSE.md</d:LicenseUrl>

**Comment by russcam** [comment]
@skofman1 You've misunderstood me 🙂 

I'd like to know what the license is for the _package metadata_ exposed in the NuGet feeds e.g.

```xml
<?xml version="1.0" encoding="utf-8"?>
<NugetDump xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <NugetPackages>
    <FeedPackage>
      <Id>NEST</Id>
      <Version>5.5.0</Version>
      <Authors>Elastic and contributors</Authors>
      <Copyright>2014-2017 Elasticsearch BV</Copyright>
      <Created>2017-07-21T07:25:27.157</Created>
      <Dependencies>Elasticsearch.Net:[5.5.0, 6.0.0):net45|Newtonsoft.Json:[10.0.0, 11.0.0):net45|Elasticsearch.Net:[5.5.0, 6.0.0):net46|Newtonsoft.Json:[10.0.0, 11.0.0):net46|NETStandard.Library:[1.6.0, ):netstandard1.3|System.Collections.Specialized:[4.3.0, ):netstandard1.3|System.Reflection.TypeExtensions:[4.3.0, ):netstandard1.3|System.Linq.Queryable:[4.0.1, ):netstandard1.3|Elasticsearch.Net:[5.5.0, 6.0.0):netstandard1.3|Newtonsoft.Json:[10.0.0, 11.0.0):netstandard1.3</Dependencies>
      <Description>Strongly typed interface to Elasticsearch. Fluent and classic object initializer mappings of requests and responses. Uses and exposes Elasticsearch.Net</Description>
      <DownloadCount>1319784</DownloadCount>
      <GalleryDetailsUrl>https://www.nuget.org/packages/NEST/5.5.0</GalleryDetailsUrl>
      <IconUrl>https://raw.githubusercontent.com/elastic/elasticsearch-net/master/build/nuget-icon.png</IconUrl>
      <IsLatestVersion>true</IsLatestVersion>
      <IsAbsoluteLatestVersion>false</IsAbsoluteLatestVersion>
      <IsPreRelease>false</IsPreRelease>
      <Language />
      <LastUpdated>2017-07-21T07:25:27.157</LastUpdated>
      <Published>2017-07-21T07:25:27.157</Published>
      <PackageHash>KWETb7Y4r5YYvEJALBZgYUhnUUgHZdt6DmDWbu1kKpVEM6o8PXOkC3/Q/D0npLue6UKZDT6FWX8ygZobnbK0Qw==</PackageHash>
      <PackageHashAlgorithm>SHA512</PackageHashAlgorithm>
      <PackageSize>5304128</PackageSize>
      <ReportAbuseUrl>https://www.nuget.org/packages/NEST/5.5.0/ReportAbuse</ReportAbuseUrl>
      <ReleaseNotes>See https://github.com/elastic/elasticsearch-net/releases/tag/5.5.0</ReleaseNotes>
      <RequireLicenseAcceptance>false</RequireLicenseAcceptance>
      <Summary>Strongly typed interface to Elasticsearch. Fluent and classic object initializer mappings of requests and responses. Uses and exposes Elasticsearch.Net</Summary>
      <Tags>elasticsearch elastic search lucene nest</Tags>
      <Title>NEST</Title>
      <VersionDownloadCount>425</VersionDownloadCount>
      <MinClientVersion />
      <LicenseUrl>https://github.com/elastic/elasticsearch-net/blob/master/license.txt</LicenseUrl>
      <LicenseNames />
      <LicenseReportUrl />
      <DownloadUrl>https://www.nuget.org/api/v2/package/NEST/5.5.0</DownloadUrl>
      <ProjectUrl>https://github.com/elastic/elasticsearch-net</ProjectUrl>
      <LastEdited>1901-01-01T00:00:00</LastEdited>
    </FeedPackage>
    <FeedPackage>
      <Id>Newtonsoft.Json</Id>
      <Version>10.0.3</Version>
      <Authors>James Newton-King</Authors>
      <Copyright>Copyright © James Newton-King 2008</Copyright>
      <Created>2017-06-18T02:10:29.603</Created>
      <Dependencies>::net20|::net35|::net40|::net45|NETStandard.Library:[1.6.1, ):netstandard1.0|Microsoft.CSharp:[4.3.0, ):netstandard1.0|System.ComponentModel.TypeConverter:[4.3.0, ):netstandard1.0|System.Runtime.Serialization.Primitives:[4.3.0, ):netstandard1.0|NETStandard.Library:[1.6.1, ):netstandard1.3|Microsoft.CSharp:[4.3.0, ):netstandard1.3|System.ComponentModel.TypeConverter:[4.3.0, ):netstandard1.3|System.Runtime.Serialization.Primitives:[4.3.0, ):netstandard1.3|System.Runtime.Serialization.Formatters:[4.3.0, ):netstandard1.3|System.Xml.XmlDocument:[4.3.0, ):netstandard1.3|::portable-net40+sl5+win8+wp8+wpa81|::portable-net45+win8+wp8+wpa81</Dependencies>
      <Description>Json.NET is a popular high-performance JSON framework for .NET</Description>
      <DownloadCount>73213422</DownloadCount>
      <GalleryDetailsUrl>https://www.nuget.org/packages/Newtonsoft.Json/10.0.3</GalleryDetailsUrl>
      <IconUrl>http://www.newtonsoft.com/content/images/nugeticon.png</IconUrl>
      <IsLatestVersion>true</IsLatestVersion>
      <IsAbsoluteLatestVersion>true</IsAbsoluteLatestVersion>
      <IsPreRelease>false</IsPreRelease>
      <Language />
      <LastUpdated>2017-06-18T02:10:29.603</LastUpdated>
      <Published>2017-06-18T02:10:29.603</Published>
      <PackageHash>wBM7i7i3U2WX0ecor4zdVGvgrwFUjuigBoHKiL+nH39fCOpCPjBY3RDqJM32edvdyTAVdjzlccHsg41+/+zpSA==</PackageHash>
      <PackageHashAlgorithm>SHA512</PackageHashAlgorithm>
      <PackageSize>2066867</PackageSize>
      <ReportAbuseUrl>https://www.nuget.org/packages/Newtonsoft.Json/10.0.3/ReportAbuse</ReportAbuseUrl>
      <ReleaseNotes />
      <RequireLicenseAcceptance>false</RequireLicenseAcceptance>
      <Tags>json</Tags>
      <Title>Newtonsoft.Json</Title>
      <VersionDownloadCount>629140</VersionDownloadCount>
      <MinClientVersion>2.12</MinClientVersion>
      <LicenseUrl>https://raw.github.com/JamesNK/Newtonsoft.Json/master/LICENSE.md</LicenseUrl>
      <LicenseNames />
      <LicenseReportUrl />
      <DownloadUrl>https://www.nuget.org/api/v2/package/Newtonsoft.Json/10.0.3</DownloadUrl>
      <ProjectUrl>http://www.newtonsoft.com/json</ProjectUrl>
      <LastEdited>1901-01-01T00:00:00</LastEdited>
    </FeedPackage>
  </NugetPackages>
</NugetDump>
``` 

Is this _package metadata_ covered under the same Apache 2 license as the source code?

**Comment by skofman1** [comment]
@anangaur , do you know?
